cmd/host/config_windows.go

```diff
@@ -71,20 +71,23 @@ func validateSubnet(s string) (*subnet, error) {
 
 func gatewayIP(ip net.IP) string {
 	// Gateway is always x.x.x.1
-	return net.IPv4(ip[0], ip[1], ip[2], 1).String()
+	gatewayLastBit := 1
```

Comment:
Perhaps you want this as a `const` elsewhere? The benefit of `const` is that it'll be an untyped int, so you don't need the cast to `byte`.